app/jobs/create_new_device_alert_job.rb

```diff
@@ -8,7 +8,7 @@ def perform(now)
     User.where(
       sql_query_for_users_with_new_device,
       tvalue: now - IdentityConfig.store.new_device_alert_delay_in_minutes.minutes,
-    ).find_each do |user|
+    ).limit(2_000).find_each(batch_size: 100) do |user|
```

Comment:
wont this create a backlog of users unless we expect to only ever get 2k users. or is this assuming were gonna have less than 2k users in the ideal state?

Reply:
alright!